docs/INSTALL.rst

```diff
@@ -48,13 +48,44 @@ This will take some time to complete, as it will install and configure a
 complete image of h and all its direct dependencies.
 
 
+Configuring container dependencies
+----------------------------------
+
+At a minimum, you will need:
+
+-  a PostgreSQL database
+-  an ElasticSearch server with the ICU analysis plugin enabled
+-  an nsqd server
+-  a Redis server
+-  a mailer
+
+One option is to containerise these services, although you should investigate
+for yourself if this is a sensible approach for your environment.
+
+For example. We choose to run ElasticSearch, nsqd, and Redis in containers. We
+also use an `ambassador container`_ to point to our mailserver::
+
+    $ docker run -d --name elasticsearch nickstenning/elasticsearch-icu
+    $ docker run -d --name nsqd --expose 4150 --expose 4151 nsqio/nsq /nsqd
+    $ docker run -d --name redis redis
+    $ docker run -d --name mail --expose 25 -e MAIL_PORT_25_TCP=tcp://smtp.mydomain.com:25 svendowideit/ambassador
```

Comment:
OK @tilgovi, but which docs? What/where should I read about how to use `--env-file`. Even the updated docs doesn’t mention it.